packages/mux-player/src/index.ts

```diff
@@ -786,9 +786,6 @@ class MuxPlayerElement extends VideoApiElement {
    * Set the beacon collection domain. Used by Mux Data.
    */
   set beaconCollectionDomain(val: string | undefined) {
-    // don't cause an infinite loop
```

Comment:
question(blocking): While this may not cause an infinite loop, doesn't this mean we're triggering re-renders unnecessarily (aka when no change occurred) now?

Reply:
yes that's a valid point, just a small thought. 

should we gate that in the property or in the attributeChangedCallback? 
makes more sense in the callback I think because attributes can always still be set and callback runs each time on a set

Reply:
agree that feels like the "right" place to do the check, but I think that should be a separate PR since it effects all attributes and may introduce a regression, since there are potential "externally set" state vs. "internally set" state considerations there.